plugins/org.eclipse.sirius.tests.swtbot/src/org/eclipse/sirius/tests/swtbot/sequence/HeaderSequenceDiagramTests.java

```diff
@@ -85,7 +87,7 @@ protected void onSetUpAfterOpeningDesignerPerspective() throws Exception {
     /**
      * Test if header is correctly renamed after the lifeLine renaming.
      */
-    public void testRenameLifeLine() {
+    public void _testRenameLifeLine() {
```

Comment:
Why have you deactivated certain tests in this file? They work for me (except for one, but it doesn't seem to have anything to do with colors). Did you forget to reactivate it?

Reply:
I forgot to reactive ;)